src/project/internal/projectuiactions.cpp

```diff
@@ -86,6 +86,13 @@ const UiActionList ProjectUiActions::m_actions = {
              TranslatableString("action", "Publish to MuseScore.com…"),
              IconCode::Code::CLOUD_FILE
              ),
+    UiAction("file-publish-audio",
+             mu::context::UiCtxAny,
+             mu::context::CTX_ANY,
+             TranslatableString("action", "Publish to A&udio.com…"),
```

Comment:
We must be careful about `&u` in this string. We also use it in `"Save to clo&ud…"`, so the `"Publish to A&udio.com…"` item cannot be in the same menu as the `"Save to clo&ud…"` item. Currently it looks like it isn't, so there is no problem, but we should just keep it in mind.